libr/bin/format/elf/elf.h

```diff
@@ -18,6 +18,23 @@
 #define R_BIN_ELF_ALL_SYMBOLS (R_BIN_ELF_SYMTAB_SYMBOLS | R_BIN_ELF_DYNSYM_SYMBOLS)
 #define ELFOBJ struct Elf_(r_bin_elf_obj_t)
 
+#if R_BIN_ELF64
+#define WORDSIZE 0x8
```

Comment:
Shouldn't these be something like `R_BIN_ELF_WORDSIZE` when they are in the header?